Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/TargetsTests/TargetsTestsUtils.cs

```diff
@@ -87,15 +87,19 @@ public void CreateCaptureDataTargetsFile(string directory, string afterTargets)
             // The SimpleXmlLogger has special handling to extract the data
             // from the message and add it to the BuildLog.
             string xml = Properties.Resources.CaptureDataTargetsFileTemplate;
-            xml = string.Format(xml, afterTargets);
-
-            // We're using :: as a separator here: replace it with whatever
-            // the logger is using as a separator
-            xml = xml.Replace("::", SimpleXmlLogger.CapturedDataSeparator);
+            CreateCaptureTargetsFile(directory, string.Format(xml, afterTargets));
+        }
 
+        public string CreateCaptureTargetsFile(string directory, string xmlTargets)
+        {
+            // Most of the tests above want to check the value of build property or item group after a target has been executed. However, this information is not available through the buildlogger interface.
```

Comment:
this comment is still present in lines 82-88 - maybe remove it from there?

or keep only what's necessary - mentioning `SimpleXmlLogger` in lines 87-88 probably isn't needed. Not sure for the rest.